server/views/outcomes/casesInProgress.njk

```diff
@@ -32,6 +105,24 @@
     controls: "outcomes"
   }) }}
 
-  <p class="govuk-body">{{ "There are no matching cases." if params.filtersApplied else "No case list information available." }}</p>
+  {% if displayFilters %}
+    {%- from "components/filter/macro.njk" import pacFilter -%}
+    {{ pacFilter({
+      filters: params.filters,
+      hiddenInputs: params.sorts
+    }) }}
+  {% endif %}
+
+  {% if tableData.rows | length %}
+    {% block table %}
+      {%- from "govuk/components/table/macro.njk" import govukTable -%}
+      {{ govukTable(tableData) }}
+    {% endblock %}
+  {% else %}
+    {% if params.filtersApplied %}
+      <span class="govuk-heading-m">0 cases</span>
+    {% endif %}
+    <p class="govuk-body">{{ "There are no matching cases." if params.filtersApplied else "No case list information available." }}</p>
```

Comment:
Is this mentioned in any of the designs? This is probably from case-list?
Please find the right design and update.

Same for cases to result tab

Reply:
There isn't any designs in regarding this, but I don't see why it wouldn't behave this way? E.g. If the user selected and applied filters which returned 0 cases

Reply:
I agree this is a very valid scenario and it should be catered to. What I meat was if you followed the screen design from the design team. I was just asking to refer to designs provided by design team rather than using the case-list ones or agree that it will be same as case-list page's design.

Reply:
Here is the screen / message for this scenario
<img width="1258" alt="image" src="https://github.com/ministryofjustice/prepare-a-case/assets/98027245/3fea0cc7-4b92-48c7-9ffb-2e8a68651718">